From 11e342706a3e99b7a08ea75c2d0cdb50d3248c25 Mon Sep 17 00:00:00 2001
From: Sowjanya vardhineni <quic_svardhin@quicinc.com>
Date: Fri, 29 Nov 2024 12:06:45 +0530
Subject: [PATCH] QSDK: DP: wifi: ath12k: Increasing HTT_PPDU_STATS_MAX_USERS
 from 8 to 37

Currently, in ath12k HTT_PPDU_STATS_MAX_USERS count is 8,
So that getting warning HTT PPDU STATS event has
unexpected num_users 8.

Hence, to fix increasing the HTT_PPDU_STATS_MAX_USERS count from 8 to 37
by refferring in ath11k.

Patch-dependency: 069-ath11k-add-HE-stats-in-peer-stats.patch

Signed-off-by: Sowjanya vardhineni <quic_svardhin@quicinc.com>
---
 drivers/net/wireless/ath/ath12k/dp.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/wireless/ath/ath12k/dp.h b/drivers/net/wireless/ath/ath12k/dp.h
index 82e3b3e..4af36de 100644
--- a/drivers/net/wireless/ath/ath12k/dp.h
+++ b/drivers/net/wireless/ath/ath12k/dp.h
@@ -1984,7 +1984,7 @@ struct htt_ppdu_user_stats {
 	struct htt_ppdu_stats_cmpltn_flush cmpltn_flush;
 };
 
-#define HTT_PPDU_STATS_MAX_USERS	8
+#define HTT_PPDU_STATS_MAX_USERS	37
 #define HTT_PPDU_DESC_MAX_DEPTH	16
 
 struct htt_ppdu_stats {
-- 
2.34.1

